doc/BUILD.md

```diff
@@ -68,30 +68,20 @@ Some package managers distribute a version of `rustup`. Failing that, you can al
 
 ## Building the project
 
-Once you've got Cargo, building the project is a simple `cargo` invocation away.
+To build the whole wallet including the GUI, you'll need to install its [build and runtime
+dependencies](https://github.com/wizardsardine/liana/tree/master/gui#dependencies) first.
 
-To only build the daemon, run it from the root of the repository:
-```
-$ cargo build --release
-```
-The `lianad` and `liana-cli` binaries will be in the `target/` directory at the root of the
-repository:
-```
-$ ls target/release/
-build  deps  examples  incremental  liana-cli  liana-cli.d  lianad  lianad.d  libliana.d  libliana.rlib
-```
+Then run:
 
-To build the whole wallet including the GUI, you'll need to install its [build and runtime
-dependencies](https://github.com/wizardsardine/liana/tree/master/gui#dependencies) first. Then run
-the same command as above within the [`gui/`](../liana-gui/) folder present at the root of the repository:
 ```
-$ cd liana-gui/
 $ cargo build --release
 ```
-The `liana-gui` binary will be in the root `target/` folder:
+The `lianad`, `liana-cli` and `liana-gui` binaries will be in the `target/` directory at the root of the
+repository as the exception for windows that only `liana-gui` will be built
```

Comment:
I would change to:
"...at the root of the repository, except if using Windows in which case only `liana-gui` will be built"